tinode-db/main.go

```diff
@@ -225,11 +225,20 @@ func main() {
 		// Upgrade DB from one version to another.
 		err = store.UpgradeDb(config.StoreConfig)
 		if err == nil {
-			log.Println("Database successfully upgraded")
+			log.Println("Database successfully upgraded. All done.")
```

Comment:
Maybe `os.Exit(0)` and `os.Exit(1)` would be more idiomatic.

Reply:
Do you mean checking for the exit code instead of the magic string?

Reply:
Yes. Sorry for not making myself clear.

Reply:
Done.